public/timing.js

```diff
@@ -2,8 +2,6 @@ window.addEventListener("load", function(){
   window.dailyFeed = document.getElementById("dailyFeed");
 
   displayPosts();
-
- 	setInterval(displayPosts, 10000);
```

Comment:
do we need to add this line back in for master? I know we all hate it sufficiently by now.